biolink/pieces-parts/rank-regulators.rkt

```diff
@@ -315,11 +318,11 @@ function) because that edge is the one that has the directional predicates.
                                (concept->curie/name g)
                                (lambda (v)
                                  (cond
-                                   [(assoc (car new-val) v) => (lambda (a) 
-                                                                 (unless (empty? pubmeds)
-                                                                   (set-add (set-remove v a)
-                                                                            (list (car a)
-                                                                                  (set-union (cadr a) pubmeds)))))]
+                                   [(and (not (empty? pubmeds))
```

Comment:
What's the problem with an empty pubmeds?

Reply:
All the inside of that cond statement does is add the pubmed ids in the variable pubmeds to the provenance that's being recorded, so it would be a waste to execute the inside statements if pubmeds is empty.

Reply:
But is it the right thing to do the else case here if the pubmed is empty?

Reply:
Oh, I see. I think the way I would try to fix that tho is to write what is in the deleted lines from the last commit, but those caused an error, so I'll look into how to fix it.

Reply:
I think the simplest way is to not treat pubmeds empty so specially, maybe only save the set-union within the first (now not pubmed specific) branch. Does it make sense?

Reply:
But I think you can rely on racket not being wasteful in that case, so I would just remove the pubmed logic entirely.

Reply:
Yeah that makes sense.